nym-wallet/src/components/Send/SendInputModal.tsx

```diff
@@ -52,28 +65,124 @@ export const SendInputModal = ({
   const [isValid, setIsValid] = useState(false);
   const [memoIsValid, setMemoIsValid] = useState(true);
   const [feeAmountIsValid, setFeeAmountIsValid] = useState(true);
+  const [addressIsValid, setAddressIsValid] = useState(false);
+  const [errorAmount, setErrorAmount] = useState<string | undefined>();
+  const [errorFee, setErrorFee] = useState<string | undefined>();
+  const [noAccount, setNoAccount] = useState(false);
+
+  useEffect(() => {
+    if (!balance || balance === '0' || parseFloat(balance) === 0) {
+      setNoAccount(true);
+    } else {
+      setNoAccount(false);
+    }
+  }, [balance]);
```

Comment:
With such simple state/props conditions, it is better to avoid unnecessary `useEffect`:
```
// component root scope
const noAccount = !balance || balance === '0' || parseFloat(balance) === 0;
```

Reply:
Nice one, changed this as well